src/app/accordion/accordion.demo.html

```diff
@@ -36,22 +36,21 @@ <h2 class="fieldset-title">Accordion</h2>
   <soho-accordion-header [isExpanded]="true">Sort By</soho-accordion-header>
   <soho-accordion-pane>
     <div class="accordion-content">
-      <div class="radio-group">
-        <input type="radio" class="radio" name="sort" id="sort-recent" value="recent">
-        <label for="sort-recent" class="radio-label">Recently Added</label>
+      <div class="field">
+        <input soho-radiobutton [(ngModel)]="model.sortBy" id="sort-recent" type="radio" name="sort" value="sort-recent" />
```

Comment:
doesn't `soho-radiobutton.component.ts` already add the type through a property binding?

Reply:
aww yea i can remove the redundancy from radio button page, i just copied it over and changed the values